wizard/WizardPasswordUI.qml

```diff
@@ -42,19 +42,22 @@ ColumnLayout {
 
       wizard.nextButton.enabled = passwordItem.password === retypePasswordItem.password
 
-      // scorePassword returns value from 0 to... lots
-      var strength = walletManager.getPasswordStrength(passwordItem.password);
-      // consider anything below 10 bits as dire
-      strength -= 10
-      if (strength < 0)
-          strength = 0
-      // use a slight parabola to discourage short passwords
-      strength = strength ^ 1.2 / 3
-      // mapScope does not clamp
-      if (strength > 100)
-          strength = 100
-      // privacyLevel component uses 1..13 scale
-      privacyLevel.fillLevel = Utils.mapScope(1, 100, 1, 13, strength)      
+      // TODO: password strength meter segfaults on Android.
```

Comment:
Any stack trace for this ?

Reply:
Initially reported by @MoroccanMalinois, do you have a stack trace for this? I don't have access to any android devices at the moment.